sensu/config.go

```diff
@@ -61,9 +76,10 @@ func NewConfigFromFlagSet(flagset *configFlagSet) (*Config, error) {
 		if err := json.Unmarshal(buf, &cfg.config); err != nil {
 			return nil, err
 		}
+		return &cfg, nil
```

Comment:
Would it be possible to replace the logic inside this `if` block with a call to `NewConfigFromFile` in order to avoid code duplication?